x/subspaces/keeper/msg_server.go

```diff
@@ -82,7 +82,7 @@ func (k msgServer) EditSubspace(goCtx context.Context, msg *types.MsgEditSubspac
 	}
 
 	// Check the permission to edit
-	if !k.HasPermission(ctx, msg.SubspaceID, signer, types.PermissionChangeInfo) {
+	if !k.HasPermission(ctx, msg.SubspaceID, 0, signer, types.PermissionChangeInfo) {
```

Comment:
Instead using 0 directly, can we usa a constant like `DefaultSection` or `RootSection`?
I think it's more expressive and easier to understand at first sight

Reply:
Done :+1: